src/App.vue

```diff
@@ -431,21 +424,15 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
     this.$root.$off('go-to-dashboard')
   }
 
-  /** Called when profile is ready -- we can now init app. */
-  @Watch('profileReady')
-  private async onProfileReady (val: boolean): Promise<void> {
```

Comment:
Instead of being triggered by the signin component, this function is now triggered by a route change (same as Create UI).